app/javascript/components/batch_update/components/__snapshots__/SearchForm.spec.js.snap

```diff
@@ -1257,6 +1258,7 @@ exports[`"edit artworks" button renders an enabled edit button if there are sele
   </div>
   <button
     className="  c4"
+    disabled={undefined}
```

Comment:
I see this is a test but this feels really weird to me. Why specify `disabled={undefined}`, what does that do? Why not leave `disabled` out entirely?

Reply:
Here's the [chapter and verse](http://w3c.github.io/html/infrastructure.html#sec-boolean-attributes) but tldr html5 boolean attributes are true merely by being present — no value need be supplied.

Reply:
(Oh, and also more pertinently, this code is auto-generated by Jest's snapshot functionality, it was not written by human hands 😛 )

Reply:
https://jestjs.io/docs/en/snapshot-testing

Reply:
lol right of course it was a snapshot, thanks. I was looking for `inline_snapshot` to see if this was contained in a spec but I didn't actually check the file path. 😇